main.go

```diff
@@ -259,6 +259,7 @@ func main() {
 		fmt.Print(out)
 
 	case "show":
+		teller.SortByProviderName()
```

Comment:
To make the main.go with less functionality and keep it only for handling CLI commands, I think it better to move it to  `PrintEnvKeys` function.
You will keep main.go for CLI command manager only + you will make sure that any time that someone in the code wants to print the envKeys it will always show it sorted

Reply:
@kaplanelad 
Agree, changed and added a test for the output